portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,27 @@
+package com.google.sps.data;
+
+import java.util.Date;
+
```

Comment:
https://google.github.io/styleguide/javaguide.html#s7.3-javadoc-where-required -- use javadoc for all public classes